packages-ts/gauntlet-terra-contracts/codeIds/bombay-testnet.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
This just needed renamed, not deleted right?

Reply:
oops. I had renamed it but git add didn't automatically pick it up